spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchQueryScan.java

```diff
@@ -93,6 +93,11 @@ class SparkBatchQueryScan extends SparkScan implements SupportsRuntimeFiltering
     }
   }
 
+  @Override
```

Comment:
I would separate these changes out as they are not related to this PR

Reply:
Removed.